### PR TITLE
Add multi-language support

### PR DESCRIPTION
**Pull request by jabby**

### CODE REVIEW

File: lib/generate-index.js [hunk 1]
```diff
@@ -1,6 +1,28 @@
 'use strict'
 
 const lunr = require('lunr')
+require("lunr-languages/lunr.stemmer.support")(lunr)
```
Comment (Mogztter) [inline lib/generate-index.js:4]: We should probably require/load them on demand no? If `DOCSEARCH_LANGS` is not defined there's no extra work needed.

Comment (jabby) [reply lib/generate-index.js:4]: Yes indeed.

File: lib/generate-index.js [hunk 2]
```diff
@@ -102,9 +103,30 @@ function generateIndex (playbook, pages, contentCatalog, env) {
...
 
+
+  const languages = env.DOCSEARCH_LANGS ?
+                    env.DOCSEARCH_LANGS.split(',') :
+                    ['en']
+
+  languages.forEach(language => {
+
+    if (language === 'ja') {
+      // require('../tinyseg')(lunr) // needed for Japanese Support (manually added in https://github.com/MihaiValentin/lunr-languages repository)
```
Comment (Mogztter) [inline lib/generate-index.js:114]: I'm not sure I understand this comment. Do we need to do something special to support Japanese?
I believe that `tinyseg.js` is part of the `lunr-languages` package so we could do the following no?

```js
require('lunr-languages/tinyseg')(lunr)
```

File: lib/generate-index.js [hunk 3]
```diff
@@ -102,9 +103,30 @@ function generateIndex (playbook, pages, contentCatalog, env) {
...
+    }
+
+    if (language === 'th') {
+      // require('../wordcut'); // needed for Thai support (manually added in https://github.com/MihaiValentin/lunr-languages repository)
```
Comment (Mogztter) [inline lib/generate-index.js:118]: ditto

File: test/test.js [hunk 4]
```diff
@@ -6,6 +6,12 @@ chai.use(dirtyChai)
 const generateIndex = require('../lib/index')
 const mockContentCatalog = require('./mock-content-catalog.js')
 
+const lunr = require('lunr')
+const lunr2 = require('lunr')
```
Comment (Mogztter) [inline test/test.js:10]: `require('lunr')` will cache the instance, so unless I'm missing something, it's effectively the same as:

```js
const lunr = require('lunr')
const lunr2 = lunr
```

Just did a quick test and the tests suite is working just fine if I remove `lunr2` and use `lunr` instead.

**Comment by Mogztter** [comment]
@jabby I rebased on master and did a few changes, let me know what you think.

**Comment by Mogztter** [comment]
Could you please document this feature on the README in the "Configuration" section?

**Comment by jabby** [comment]
@Mogztter of course. I will update the PR next saturday.
By the way, I love your refactoring. It make sense. 👍

**Comment by Mogztter** [comment]
Perfect thanks :ok_hand: 
I'm glad you like it! Feel free to make any adjustments necessary.

**Comment by jabby** [comment]
It tooks me more time than wanted. but I added the missing documentation. 

Feel free to update if something is wrong.

**Comment by Mogztter** [comment]
Thank you, it looks good :ok_hand:

**Pull request merged**